DFC.ServiceTaxonomy.GraphSync/Handlers/Orchestrators/SyncOrchestrator.cs

```diff
@@ -26,16 +31,22 @@ public class SyncOrchestrator : Orchestrator, ISyncOrchestrator
             IContentDefinitionManager contentDefinitionManager,
             INotifier notifier,
             IGraphCluster graphCluster,
-            IPublishedContentItemVersion publishedContentItemVersion,
             IServiceProvider serviceProvider,
-            ILogger<SyncOrchestrator> logger)
-            : base(contentDefinitionManager, notifier, logger)
+            ILogger<SyncOrchestrator> logger,
+            IOptionsMonitor<EventGridConfiguration> eventGridConfiguration,
+            IEventGridContentClient eventGridContentClient,
+            ISyncNameProvider syncNameProvider,
+            IPublishedContentItemVersion publishedContentItemVersion,
+            IPreviewContentItemVersion previewContentItemVersion,
+            INeutralEventContentItemVersion neutralEventContentItemVersion)
+            : base(contentDefinitionManager, notifier, logger, eventGridConfiguration, eventGridContentClient, syncNameProvider, publishedContentItemVersion, previewContentItemVersion, neutralEventContentItemVersion)
         {
             _graphCluster = graphCluster;
             _publishedContentItemVersion = publishedContentItemVersion;
             _serviceProvider = serviceProvider;
         }
 
+        //Doesn't this and the other similar methods on these orchestrators return false if it failed or was blocked?
```

Comment:
You're right, would you mind changing the comments?

Reply:
Done!